### PR TITLE
Allow multiple named parameter in single path part

### PR DESCRIPTION
**Pull request by MugeSo**
This PR allows `/{foo}-{bar}/{file}.{ext}`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -399,7 +399,9 @@ function byProperty(property, value) {
 function byRoute(a, b) {
   if(isDynamicRoute(a.path) && !isDynamicRoute(b.path)) return 1;
   if(!isDynamicRoute(a.path) && isDynamicRoute(b.path)) return -1;
-  return a.path.localeCompare(b.path);
+
+  // invert compare to keep that /{foo} does not beat /{foo}.{bar}
+  return -1*a.path.localeCompare(b.path);
```
Comment (jsdevel) [inline index.js]: space around operators e.g. `-1 * a.path......`

**Comment by MugeSo** [comment]
> Can you also add this to the README? 

I don't know which section should I write it in.

**Comment by jsdevel** [comment]
> I don't know which section should I write it in.

How about in a new description under [this heading](https://github.com/kogosoftwarellc/express-openapi#operation-parameters).  There's currently just a bullet point underneath that.  We should probably have a sub heading titled `Path Configuration` above the existing bullet point.

**Comment by jsdevel** [comment]
On second thought, how about a new `Path Parameters` section above `Operation Parameters` that explains how `{param}` works in the filename as well as an example of multiple params in a path part.

**Comment by jsdevel** [comment]
@MugeSo any updates?

**Comment by jsdevel** [comment]
@MugeSo if you can add documentation later I'd appreciate it.  Thanks for the change!

**Pull request merged**